Sources/General/FileHelper.swift

```diff
@@ -34,19 +31,23 @@ final class FileHelper {
                      contentModificationDate: values?.contentModificationDate)
     }
 
-    func createDirectory(at path: String) -> Bool {
+    func createDirectory(at path: String) throws {
+        try fileManager.createDirectory(atPath: path,
+                                        withIntermediateDirectories: true,
+                                        attributes: nil)
+    }
+
+    func createDirectoryIfPossible(at path: String) -> Bool {
         do {
-            try fileManager.createDirectory(atPath: path,
-                                            withIntermediateDirectories: true,
-                                            attributes: nil)
+            try createDirectory(at: path)
         }
         catch {
             return false
         }
         return true
     }
 
-    func createDirectory(at url: URL) -> Bool {
-        createDirectory(at: url.path)
+    func createDirectoryIfPossible(at url: URL) -> Bool {
+        createDirectoryIfPossible(at: url.path)
     }
```

Comment:
These functions are using in functions which are marked as `throws`. So it looks way easier to use 
```
try fileHelper.createDirectoryIfPossible(at: url)
```
instead of
```
if fileHelper.createDirectoryIfPossible(at: url) == false {
    throw Error.write(url)
}
```
Or am I missing something?

Reply:
`Setup` provides its own error descriptions. FileManager error will contain too specific technical information and can be not printing pretty by `ArgumentParser`

Reply:
https://github.com/rosberry/general/pull/1#issuecomment-690143223